### DISCUSSION TITLE
Orc8r Database Visualization

### DISCUSSION THREAD

**Discussion by hcgatewood** [magma/magma#7845] [Show and tell]
This is a visualization of the Orc8r database tables, a few commits before the v1.6 release

![orc8r_tables](https://user-images.githubusercontent.com/8029544/123882940-825f0680-d90d-11eb-974e-c126e27e2544.png)

**Comment by karthiksubraveti** [comment]
Hey @hcgatewood , this is great. Thanks for the visualization. One thing we lack is solid documentation on what entities/associations/states are. Can you add this in our developer documentation on how one should get started with adding entities to Orc8r.
